app/src/main/res/layout/sheet_phone_mask.xml

```diff
@@ -62,7 +62,10 @@
       android:drawableStart="@drawable/ic_phone_locked_24dp"
       android:text="@string/phonemasking_call_secure"
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button1.White100"
-      app:drawableTint="@color/white" />
+      app:drawableTint="@color/white"
+      android:visibility="gone"
+      android:clickable="false"
```

Comment:
A view that is invisible can still be navigated to and clicked using physical keyboards/key commands. Since this is an important feature that must be disabled, I decided to set the `clickable` to `false` as well.

Reply:
@vinaysshenoy but the view's visibility is `gone` here, which means it isn't even going to be rendered on the screen. So the `clickable` property is redundant here.

Reply:
I suppose I am overthinking this, though. 😅